swagger_templates/python/README.diff

```diff
@@ -0,0 +1,83 @@
+0a1,14
+> # Python Language Bindings
+> 
+> This repository is part of the Isilon SDK, which is an evolving package of
+> documents and files. This README describes how to install and configure Python
+> language bindings in order to access the OneFS API on an Isilon cluster from a
+> Python script. You can use the OneFS API to automate the configuration,
+> maintenance, and monitoring of your Isilon cluster.
+> 
+> * For OneFS API reference documents, discussions, and blog posts, refer to the
+> [OneFS SDK Info Hub](https://community.emc.com/docs/DOC-48273).
+> * To browse the Isilon InsightIQ statistics API, refer to the
+> [Stat Key Browser](https://github.com/isilon/isilon_stat_browser.git)
+> GitHub repository.
+> 
+25c39
+< If the python package is hosted on Github, you can install directly from Github
+---
+> Install the package from the Python Package Index
+28c42
+< pip install git+https://github.com/{{{gitUserId}}}/{{{gitRepoId}}}.git
+---
+> pip install {{{packageName}}}
+30,44c44
+< (you may need to run `pip` with root permission: `sudo pip install git+https://github.com/{{{gitUserId}}}/{{{gitRepoId}}}.git`)
+< 
+< Then import the package:
+< ```python
+< import {{{packageName}}}
+< ```
+< 
+< ### Setuptools
+< 
+< Install via [Setuptools](http://pypi.python.org/pypi/setuptools).
+< 
+< ```sh
+< python setup.py install --user
+< ```
+< (or `sudo python setup.py install` to install the package for all users)
+---
+> (you may need to run `pip` with root permission: `sudo pip install {{{packageName}}}`)
```

Comment:
It might be safer to suggest `pip install --user {{{packageName}}}` to avoid messing with system packages. I'm not sure how much we want to diverge from upstream's docs, but there is generally consensus that sudo pip is best avoided.

Reply:
I have already made some significant changes to the README template, so I am good with just removing this suggestion.